### PR TITLE
70 automatically signin user who creates signin

### PR DESCRIPTION
**Pull request by c0untingNumbers**
change user to be the person executing command, should be something like i.Member.User.ID
change span context to match execution block (use span instead of span_signinSlug)

### CODE REVIEW

**Comment by 1nv8rzim** [comment]
I will look closer when I get home. But looks like you did not change existing interaction response, so it will
cause an error because they are trying to respond to the same interaction.

**Comment by c0untingNumbers** [comment]
Yeah, I am aware of that. Was going to communicate with you about that, just confused about it.

File: commands/slash/signin.go [hunk 1]
```diff
@@ -142,6 +142,53 @@ func Signin() (*discordgo.ApplicationCommand, func(s *discordgo.Session, i *disc
...
+				Type: discordgo.InteractionResponseChannelMessageWithSource,
+				Data: &discordgo.InteractionResponseData{
+					Content: signinMessage(i.Member.User.ID, entSigninType, span.Context()),
+					Flags:   discordgo.MessageFlagsEphemeral,
+				},
+			})
+			if err != nil {
+				logging.Error(s, err.Error(), i.Member.User, span, logrus.Fields{"error": err})
+				return
+			}
```
Comment (1nv8rzim) [inline commands/slash/signin.go:190]: Message is sent to command user here

Comment (1nv8rzim) [reply commands/slash/signin.go:190]: and here: https://github.com/ritsec/ops-bot-iii/blob/70-automatically-signin-user-who-creates-signin/commands/slash/signin.go#L273-L282

Comment (1nv8rzim) [reply commands/slash/signin.go:190]: These both use the same interaction; best way to do this is just combine this into the later one

**Pull request merged**